docs/ref/files/storage.txt

```diff
@@ -115,7 +115,10 @@ The ``Storage`` class
 
     .. method:: get_created_time(name)
 
-        Returns a :class:`~datetime.datetime` of the creation time of the file.
+        Returns a :class:`~datetime.datetime` of the system’s ctime which, on
```

Comment:
This is the documentation for `Storage` but this an implementation detail of `FileSystemStorage` so I'd probably add `.. method:: get_created_time(name)` and this description under that class. I'd add a link to `os.path.getctime` so it's more clear what "system's ctime" means. (By the way, there's no need to use a right quote here, sphinx will render it appropriately).